TheMovieDb/HomeViewController.swift

```diff
@@ -42,16 +75,22 @@ class HomeViewController: UIViewController, UICollectionViewDelegate {
 }
 
 // MARK: - UICollectionView
-extension HomeViewController: UICollectionViewDataSource {
-    public func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+extension HomeViewController {
+    public override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
         movies.count
     }
-    
-    public func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        if let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "cell", for: indexPath) as? MovieCell {
-            cell.label?.text = "\(movies[indexPath.row].title)"
-            return cell
-        }
-        fatalError("Unable to dequeue subclassed cell")
+
+    public override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "cell", for: indexPath) as! MovieCell
+        cell.label?.text = movies[indexPath.row].title
+        return cell
+    }
+    override func numberOfSections(in collectionView: UICollectionView) -> Int {
+        2
```

Comment:
this is a magic number, we should know from something how many sections are present sorta like movies var